assets/css/styleguide.scss

```diff
@@ -1020,6 +1035,13 @@ $font-light: 300;
       font-size: rem(15px);
       line-height: 1.7;
     }
+
+    .usa-font-example p.usa-font-lead {
```

Comment:
@maya  It wouldn't work otherwise. Happy to change if there's a cleaner solution.